examples/rbd/plugin-deploy.sh

```diff
@@ -1,15 +1,33 @@
 #!/bin/bash
 
 deployment_base="${1}"
+shift
+kms_base="${1}"
 
-if [[ -z $deployment_base ]]; then
+if [[ -z ${deployment_base} ]]; then
```

Comment:
if you're changing this, please place it between `""` too.

Reply:
Sure, I'll add that. Thanks